### DISCUSSION TITLE
Any Open Telemetry examples for Pekko

### DISCUSSION THREAD

**Discussion by swapdonkey** [apache/pekko#1397] [General]
Trying to find some examples of open telemetry and Pekko Streams. It might be good to include some docs / examples as Open Telemetry grows in popularity.

**Comment by pjfanning** [comment]
Have a look at https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/9528

You might be able to work out how to use it based on the OpenTelemetry docs for either Pekko or Akka.

Specialised examples would be great. I'm afraid Apache Pekko is a volunteer project and people volunteering to provide us examples is how we work.
